docs/release_notes.rst

```diff
@@ -1,10 +1,16 @@
+.. _rel_notes:
 
 Release Notes
 *************
 
+
+.. _rel-narative:
+
 Narrative
 =========
 
```

Comment:
Do you want to add one here for this release?

Reply:
First draft added.